pages/dojo.md

```diff
@@ -0,0 +1,25 @@
+title: Code Dojo
+---
+
+On Thursday evening we'll be running a Python Code Dojo, based on the long-running
+[London Python Code Dojo](http://ldnpydojo.org.uk/) series, now approaching its 10th year.
+
+It's a very welcoming format where a coding challenge is set and everyone takes
+part as one of a team. It's definitely _not_ a competition, and it definitely
+_is_ a place where beginners or less experienced coders can join in.
+
+The layout of the evening is as follows:
+
+* We eat, drink and socialise
+* We choose a challenge
+* We break up into teams
+* Each team decides how to approach the challenge, both from
+  a technical and from a team perspective.
+* There's about an hour and a half of coding mayhem
+* Each team shows its code and results in front of everyone else:
+  how far they got, what they tried, what worked, what didn't.
+* We go home happy
+
+Because we can't accommodate the entire conference, you'll be sent a link
```

Comment:
I thought people were just going to turn up (rather than sign up)..? I.e. the whole point of choosing the robot game was to accommodate plenty of teams. (I may be misremembering).

Reply:
I defer to @inglesp but I think the point is that we need *some* idea of numbers (and perhaps a cap) so that we don't overflow the venue and have too much / too little pizza.

Reply:
We need to give the venue an idea of numbers for catering purposes, otherwise we'll either (a) have a repeat of potatogate, or (b) be on the hook for 1000 pizzas.

Reply:
OK... so given the nature of the challenge, we need 8 teams. If we have 5 as the maximum number for a team that gives us an upper limit of 40 attendees. In the past, we're just done a first come, first served thing and closed the door when 40 people have walked in. Does that help?

Reply:
I'm surprised to see a limit of 40 -- I thought the whole point of choosing the task you chose was to avoid having a limit.  Can we discuss in Slack?